apps/extension/src/ui/apps/dashboard/routes/Settings/Accounts/TreeItem.tsx

```diff
@@ -64,38 +64,42 @@ export const TreeAccountItem = forwardRef<HTMLDivElement, Props & { item: UiTree
     if (!account) return null
     return (
       <TreeItemWrapper {...props} ref={wrapperRef}>
-        <div
-          ref={ref}
-          className={classNames(
-            "bg-black-secondary relative flex items-center gap-8 rounded-sm border-[1px] border-transparent p-8",
-            depth > 0 && "bg-black-secondary/60"
-          )}
-          style={style}
-        >
-          <DragButton {...handleProps?.attributes} {...handleProps?.listeners} />
-          <AccountIcon
-            className="text-xl"
-            address={item.address}
-            genesisHash={account?.genesisHash}
-          />
-          <div className="flex grow flex-col gap-2 overflow-hidden">
-            <div className="flex items-center gap-2">
-              <div className="overflow-hidden text-ellipsis whitespace-nowrap">{account.name}</div>
-              <AccountTypeIcon className="text-primary" origin={account.origin} />
+        <div ref={ref} className="relative flex items-center gap-8" style={style}>
+          <div
+            className={classNames(
+              "bg-black-secondary flex flex-grow items-center gap-8 rounded-sm border-[1px] border-transparent p-8",
+              depth === 0 && "pl-24",
```

Comment:
need to add `overflow-hidden` on lines 70 and 153 to prevent the grid rows to go larger than the table

Reply:
Oh good catch!